x-pack/plugins/index_management/public/application/lib/data_streams.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DataStream } from '../../../common';
+
+export const isManagedByIngestManager = (dataStream: DataStream): boolean => {
+  return Boolean(dataStream._meta?.managed && dataStream._meta?.managed_by === 'ingest-manager');
+};
+
+export const filterDataStreams = (dataStreams: DataStream[], managed: boolean): DataStream[] => {
+  return dataStreams.filter((dataStream: DataStream) =>
+    managed ? isManagedByIngestManager(dataStream) : !isManagedByIngestManager(dataStream)
```

Comment:
This is a little hard for me to parse. If I'm understanding correctly, it looks like `managed` is only ever going to be false.

```
const filteredDataStreams = isIncludeManagedChecked
      ? dataStreams
      : filterDataStreams(dataStreams, isIncludeManagedChecked);
```

If that's correct, could the filter be simplified to this?

```
return dataStreams.filter((dataStream: DataStream) => isManagedByIngestManager(dataStream) === false)
```